ubuntu_image/helpers.py

```diff
@@ -101,6 +101,17 @@ def get_qemu_static_for_arch(arch):
         }
     return 'qemu-{}-static'.format(archs.get(arch, arch))
 
+def _debug_log_run(cmd, stdout, stderr):
```

Comment:
If we were to add this helper function, we'd need to have some test coverage for it. Otherwise the tests will be failing - could you add some unit test checking if this is called at the right time and if the function generates expected output?
Basically we want to have 100% code coverage at all times.

Reply:
@sil2100 
i have it on my todo list. 

I will soon add test for both PRs

Reply:
Just add the tests